packages/core/src/utils/ProfileUtils.ts

```diff
@@ -1,109 +0,0 @@
-/*
-* This program and the accompanying materials are made available under the terms of the
-* Eclipse Public License v2.0 which accompanies this distribution, and is available at
-* https://www.eclipse.org/legal/epl-v20.html
-*
-* SPDX-License-Identifier: EPL-2.0
-*
-* Copyright Contributors to the Zowe Project.
-*
-*/
-
-import { ImperativeConfig, IProfileLoaded, CliProfileManager, IProfile, Logger, ImperativeError } from "@zowe/imperative";
-import * as path from "path";
-import * as os from "os";
-
-/**
- * Retrieves the Zowe CLI home directory. In the situation Imperative has
- * not initialized it we mock a default value.
- * @returns {string} - Returns the Zowe home directory
- */
-export function getZoweDir(): string {
-    const defaultHome = path.join(os.homedir(), ".zowe");
-    if (ImperativeConfig.instance.loadedConfig?.defaultHome !== defaultHome) {
-        ImperativeConfig.instance.loadedConfig = {
-            name: "zowe",
-            defaultHome,
-            envVariablePrefix: "ZOWE"
-        };
-    }
-    return ImperativeConfig.instance.cliHome;
-}
```

Comment:
Curious if we need to create an equivalent `getZoweDir` method in the ProfileInfo APIs for Zowe Explorer

Please note that there is only one reference in Zowe Explorer to this method
- https://github.com/zowe/vscode-extension-for-zowe/blob/next/packages/zowe-explorer-api/src/utils/FileManagement.ts#L36-L38

```
    public static getZoweDir(): string {
        return zowe.getZoweDir();
    }
```

So we could just move it over there 😋

Reply:
I removed ProfileUtils.ts because I found no use of any of its functions (including getZoweDir). I missed ZE's use of getZoweDir. I could resurrect ProfileUtils.ts with just getZoweDir, or place getZoweDir into profileUtils.ts as you suggested. Do you think one choice is better than the other?

Either way, I think one of those changes should be made before merging this PR into next. Thanks.